pom.xml

```diff
@@ -99,21 +113,12 @@
       </plugin>
     </plugins>
   </build>
-
-  <distributionManagement>
```

Comment:
Is it safe to remove the distributionManagement section? Won't that break the releaser jobs?

Reply:
core does not have these sections which gave me some confidence it was historical artifact.  I intended to add back if necessary.